packages/collector/test/metrics/app/test.js

```diff
@@ -65,6 +65,10 @@ describe('snapshot data and metrics', function() {
         expect(findMetric(allMetrics, ['gc', 'minorGcs'])).to.exist;
         expect(findMetric(allMetrics, ['gc', 'majorGcs'])).to.exist;
 
+        const gc = aggregated.gc;
+        expect(gc).to.be.an('object');
+        expect(gc.statsSupported).to.be.true;
```

Comment:
This test tests the case where the native add-on (gcstats.js in this case) is available immediately. Unfortunately, this test never checked for the supported flag, so the tests missed the issue that this PR fixes.